README.md

```diff
@@ -204,6 +204,9 @@ The low-level API is as follows:
 
 For examples how to use them, please refer to the README of the above mentioned solc-js releases.
 
+**Note**: These low-level functions remain available for compatibility reasons.
+However, they were superseded by the `compile()` function and are no longer required in newer versions. The functions `compileSingle`, `compileMulti` and `compileCallback` are always `null` in the newer versions.
```

Comment:
Please clarify that they're `null` when you use a newer binary (might also be good to check which version specifically; was that 0.5.0?) rather than newer solc-js. This distinction might not be clear to users.

For some context: it is recommended to always use latest release of solc-js and it's supposed to be able to handle handle all the older binaries, down to 0.1.x. Might be good to mention that in the README too if it's not mentioned already.

Reply:
Yes, `0.5.0+commit.1d4f565a`.